### ISSUE TITLE
Home Assistant warning: Entity alarm_control_panel.sector implements device_state_attributes

### ISSUE THREAD

**Issue by robinsmidsrod** [gjohansson-ST/sector#45]
I'm getting this warning in my Home Assistant log:

```
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: January 8, 2022, 17:57:13 (5 occurrences)
Last logged: January 8, 2022, 17:57:13

Entity alarm_control_panel.sector (<class 'custom_components.sector.alarm_control_panel.SectorAlarmPanel'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.sector_living_room_temperature (<class 'custom_components.sector.sensor.SectorAlarmTemperatureSensor'>) implements device_state_attributes. Please report it to the custom component author.
```

Couldn't see another bug for it, so I thought it should be reported.

Home Assistant Core: 2021.12.8
Sector Alarm custom component version: 0.3.3

**Comment by gjohansson-ST** [comment]
Thanks for reporting. Already part of next release that will be out shortly.

**Comment by gjohansson-ST** [comment]
Solved in latest release v0.3.4
